BACKLOG.md

```
@@ -1,558 +1,348 @@
-**Decision**: Start with basic Changesets (TODO.md). Add categories if changelog becomes hard to read (>10 changes/release).
+# Restore specific function
+git show 3322953~1:convex/spacedRepetition.ts > /tmp/old_spacedRepetition.ts
+# Copy function from /tmp/old_spacedRepetition.ts
+```
 
-**Effort**: 2-3h | **Impact**: LOW - Polish, not functionality
+**When to rebuild**: When dashboard analytics are prioritized, rebuild with bounded queries:
+- Use `.take(limit)` instead of `.collect()`
+- Cache results in separate analytics table
+- Update incrementally on mutations
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)